macros/SpecData.json

```diff
@@ -194,6 +194,11 @@
     "url": "https://drafts.csswg.org/css-display/",
     "status": "WD"
   },
+  "CSS3 Environment Variables": {
+    "name": "CSS Environment Variables Module Level&nbsp;1",
+    "url": "https://drafts.csswg.org/css-env-1/",
+    "status": "WD"
```

Comment:
As this proposal is a draft, the value for `status` should be `Draft`

Reply:
This is now fixed, thanks!

Reply:
I think this should actually be "ED" for Editor's Draft instead.

Reply:
@Elchi3 this is an unofficial draft, so is "ED" the same?

Reply:
What does "unofficial draft" mean? It is presented as an ED and located under (I guess) an "official" domain https://drafts.csswg.org/

Reply:
Ok, the red label on the left now says "Editor's draft", indeed! ;-)